src/services/webuntis/model.js

```diff
@@ -2,13 +2,22 @@ const mongoose = require('mongoose');
 
 const { Schema } = mongoose;
 
+const timeSchema = new Schema({
+	weekday: {
+		type: Number, min: 0, max: 6, required: true,
+	},
+	startTime: { type: Number },
+	duration: { type: Number },
+	room: { type: String },
+});
+
 const webuntisMetadataSchema = new Schema({
 	datasourceId: { type: Schema.Types.ObjectId, ref: 'datasource' },
 	teacher: { type: String },
-	class: { type: String },
+	classes: [ { type: String } ],
```

Comment:
Codacy found an issue: [There should be no space after '['.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045816)